.pre-commit-config.yaml

```diff
@@ -36,7 +36,7 @@ repos:
     rev: 6.3.0
     hooks:
       - id: pydocstyle
-        name: pydocstyle --convention==google
+        name: pydocstyle --convention=google
```

Comment:
this isn't required for the upgrade but it is wrong and confused me a few times so I figured might as well sneak it in here